compactor/src/parquet_file_filtering.rs

```diff
@@ -52,49 +153,84 @@ pub(crate) fn filter_hot_parquet_files(
     // file. At the end of this function, the level 0 files are added to the end so they are sorted
     // last.
     let mut files_to_return = Vec::with_capacity(level_0.len() + remaining_level_1.len());
+    // Estimated memory bytes needed to compact returned L1 files
+    let mut l1_estimated_budget = Vec::with_capacity(level_0.len() + remaining_level_1.len());
     // Keep track of level 0 files to include in this compaction operation; maintain assumed
     // ordering by max sequence number.
     let mut level_0_to_return = Vec::with_capacity(level_0.len());
-    // Running total of the size, in bytes, of level 0 files and level 1 files for inclusion.
-    // The levels are counted up separately for metrics but added together when comparing with the
-    // max_size.
-    let mut total_level_0_bytes = 0;
-    let mut total_level_1_bytes = 0;
+    // Estimated memory bytes needed to compact returned L0 files
+    let mut l0_estimated_dudget = Vec::with_capacity(level_0.len());
```

Comment:
Typo in variable name:

```suggestion
    let mut l0_estimated_budget = Vec::with_capacity(level_0.len());
```